cpp/src/phonenumbers/phonenumbermatcher.h

```diff
@@ -101,6 +102,9 @@ class PhoneNumberMatcher {
     DONE,
   };
 
+  // API to check if the provided text_ is in UTF-8 or not.
```

Comment:
Remove "API to" and start with just "Checks if the ...."

Reply:
Done. Thanks Lara :)

Reply:
This one looks not done